### PR TITLE
Changed master to main

### PR DESCRIPTION
**Pull request by Shorla**
updated the tutorial from the use of master to main

### CODE REVIEW

**Comment by bebatut** [comment]
Thanks @Shorla. The images should also probably be updated. Do you think you could do that?

File: topics/contributing/tutorials/github-command-line-contribution/tutorial.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ In GitHub flow, there is a concept: one new feature or change = one branch.
 
 When you're working on a project, you're going to have a bunch of different features or ideas in progress at any given time – some of which are ready to go, and others which are not. Branching exists to help you manage this workflow. You should develop different features on different branches to help keep the changes you make simple and easy to review.
 
-![Divergence of a branch compared to master](../../images/PR_branches_01.png "When you create a branch in your project, you're creating an environment where you can try out new ideas. Changes you make on a branch don't affect the master branch")
+![Divergence of a branch compared to main](../../images/PR_branches_01.png "When you create a branch in your project, you're creating an environment where you can try out new ideas. Changes you make on a branch don't affect the master branch")
```
Comment (hexylena) [inline topics/contributing/tutorials/github-command-line-contribution/tutorial.md:121]:
```suggestion
![Divergence of a branch compared to main](../../images/PR_branches_01.png "When you create a branch in your project, you're creating an environment where you can try out new ideas. Changes you make on a branch don't affect the main branch")
```

**Comment by Shorla** [comment]
@hexylena yes I can. where can I find them?
I didn't see them on the page.

**Comment by lldelisle** [comment]
@Shorla All images except one have already been changed.If you could change the conclusion one (in `training-material/topics/contributing/images/`) , this would be super great.

**Review by shiltemann — APPROVED** [review]
Thanks a lot @Shorla!

**Pull request merged**